crates/configuration/src/shared/constants.rs

```diff
@@ -0,0 +1,15 @@
+pub const VALID_REGEX: &str = "Regex should be valid. ";
+pub const BORROWABLE: &str = "Must be borrowable as mutable. ";
+pub const RELAY_NOT_NONE: &str =
+    "Typestate should ensure the relaychain isn't None at this point. ";
+pub const SHOULD_COMPILE: &str = "Should compile with success. ";
+pub const INFAILABLE: &str = "Infaillible. ";
+pub const NO_ERR_DEF_BUILDER: &str = "Should have no errors for default builder. ";
+pub const RW_FAILED: &str = "Should be able to read/write - failed. ";
+pub const DEFAULT_TYPESTATE: &str = "'default' overriding should be ensured by typestate.";
+pub const VALIDATION_CHECK: &str = "Validation failed. ";
+
+pub const PREFIX_CANT_BE_NONE: &str = "Name prefix can't be None if a value exists.";
+
+pub const THIS_IS_A_BUG: &str =
+    "This is a bug please report it: https://github.com/paritytech/zombienet-sdk/issues";
```

Comment:
One thing we could agree on is having non-capitalized error message ? This is something common in Rust, wdyt ?

Reply:
I am not aware of that. If its a "thing" we should do it

Reply:
https://rust-lang.github.io/api-guidelines/interoperability.html?highlight=message#error-types-are-meaningful-and-well-behaved-c-good-err

```
The error message given by the Display representation of an error type should be lowercase without trailing punctuation, and typically concise.
```
We can take this approach :)